aspnetcore/mvc/views/tag-helpers/builtin/resources/ImageTagHelper.md

```diff
@@ -0,0 +1,50 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
+
+# ImageTagHelper
+
+By [Peter Kellner](http://peterkellner.net) 
+
+The Image Tag Helper enhances the `img` (`<img>`) tag. It requires a `src` tag as well as the `boolean` attribute `asp-append-version`.
+
+If the image source (`src`) is a static file on the host web server, a unique cache busting string is appended as a query parameter to the image source. This insures that if the file on the host web server changes, a unique request URL is generated that includes the updated request parameter. The cache busting string is a unique value representing the `SHA256` hash of the static image file.
+
+If the image source (`src`) isn't a static file, possibly a remote URL or the file doesn't exist on the server, the `<img>` tag's `src` attribute is generated with no cache busting query string parameter.
+
+## Image Tag Helper Attributes
+
+- - -
+
+### asp-append-version
+
+`asp-append-version` is the only attribute available for the Image Tag Helper.  When specified along with a `src` attribute, this Image Tag Helper is invoked.
+
+An example of a valid `img` tag helper is:
+
+```html
+<img src="~/images/asplogo.png" 
+    asp-append-version="true"  />
+```
+
+If the static file exists on the web server, typically in the directory `..wwwroot/images/asplogo.png` the generated html is:
+
+```html
+<img 
```

Comment:
Combine these two lines into one line of code between the two triple-backtick lines ... taking this from four lines to three lines.